ukis_pysat/members.py

```diff
@@ -1,11 +1,12 @@
 from enum import Enum
+
 from pydantic import BaseModel
 
 
 class Platform(Enum):
     Sentinel1 = "Sentinel-1"
     Sentinel2 = "Sentinel-2"
-    Sentinel3 = "Sentinel-3"
+

```

Comment:
I don't clearly see why we need to drop Sentinel-3.

Reply:
If we remove things like https://github.com/dlr-eoc/ukis-pysat/pull/163#discussion_r822037081 we can keep it I guess.